### PR TITLE
Define COR before usage

### PR DESCRIPTION
**Pull request by jsmpereira**

### CODE REVIEW

**Review by kse-clearhaus — APPROVED** [review]
:+1:

Otherwise the document is completely incomprehensible!

File: legislation.tex [hunk 1]
```diff
@@ -41,13 +41,13 @@ \chapter{Legislative basics}
...
+    procure proper \emph{cake}.
 
   \item \Cref{chp:circumstances} describes circumstances in which a traditional
-    offering of \emph{cake}{} is expected by the \emph{cake offering
-    responsible} (COR).  Individuals having undergone/undergoing any described
-    circumstance are referred to as \emph{cake offering responsible}. Notice
-    that \textbf{§12} is an exception.
+    offering of \emph{cake} is expected by the \emph{COR}.  Individuals having 
+    undergone/undergoing any described circumstance are referred to as
+    \emph{cake offering responsible}. Notice that \textbf{§12} is an exception.
```
Comment (hovmand) [inline legislation.tex]: Why not abbreviate it here?

**Review by hovmand — APPROVED** [review]

**Comment by noxern** [comment]
> Individuals having undergone/undergoing any described circumstance are referred to as \emph{COR}.

Shouldn't this precede the first usage of COR?

**Comment by jsmpereira** [comment]
> Shouldn't this precede the first usage of COR?

Let me  direct you to the Legal team.

**Comment by noxern** [comment]
I have just been in contact with the legal team. They assured me that Mrs José Santos Martins Pereira will correct this blunder.

**Comment by kse-clearhaus** [comment]
Do we need a stand-up session or sprint for this, or shall we just merge it??

**Comment by jsmpereira** [comment]
Yes.

**Pull request merged**